### DISCUSSION TITLE
Explorer plugin- Open directory location  in cmd, bash or powershell via context menu

### DISCUSSION THREAD

**Discussion by jjw24** [Flow-Launcher/Flow.Launcher#509] [Ideas]
**Is your feature request related to a problem? Please describe.**

Allow Explorer plugin to open via context menu the directory location in cmd, bash or powershell.

**Describe the solution you'd like**

In the Explorer plugin, when in a directory, go to context menu and have the options to open the directory location in cmd, bash or ps.

**Comment by jjw24** [comment]
@taooceros why is this moved to discussion? this is a piece we can action on and doesnt require any further discussion

**Reply by taooceros** [reply]
Sorry this shouldn't be moved.

**Reply by taooceros** [reply]
Do we want to included it in 1.8.0 or in the future?

**Reply by taooceros** [reply]
Though, it is a good example for testing the feature `create issue from discussion`.🤣

**Reply by jjw24** [reply]
> Do we want to included it in 1.8.0 or in the future?

probably not, once the blur pr is done, we can hold off on others, test and then release 1.8.0 after a day or two. This feature is not urgent

**Reply by jjw24** [reply]
> Though, it is a good example for testing the feature `create issue from discussion`.🤣

hmm the create issue doesnt feel as nicely done as the transfer issue to discussion. formatting seems to have been lost in description

**Reply by taooceros** [reply]
I think it embed a view of the discussion description.

**Reply by taooceros** [reply]
No, the format is preserved, except the white line?

**Reply by jjw24** [reply]
Yeah, doesn't matter really, just something I noticed
